autocomplete/forms.py

```diff
@@ -0,0 +1,3 @@
+from django import forms
+
+# place form definition here
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/autocomplete/forms.py
+++ b/autocomplete/forms.py
@@ -1,3 +1,3 @@
 from django import forms
 
-# place form definition here+# place form definition here
```